concepts/variadic-functions/about.md

```diff
@@ -0,0 +1,57 @@
+# About
+Functions in general accept only a fixed number of arguments. A variadic function is a function that accepts a variable number of arguments. If the last parameter of a function definition is prefixed by ellipsis ..., then the function can accept any number of arguments for that parameter.
+
+```go
+func hello(a int, b ...int) {
+}
```

Comment:
You could add `// ...` inside the function to indicate the function body.

Reply:
resolved